CMakeLists.txt

```diff
@@ -62,11 +62,9 @@ if (IS_DIRECTORY "${PROJECT_SOURCE_DIR}/.git")
     OUTPUT_STRIP_TRAILING_WHITESPACE)
 
   # Install Git pre-commit hook
-  if (NOT WIN32)
-    file(
-      COPY scripts/pre-commit scripts/commit-msg
-      DESTINATION "${PROJECT_SOURCE_DIR}/.git/hooks")
-  endif ()
+  file(
+    COPY scripts/pre-commit scripts/commit-msg
+    DESTINATION "${PROJECT_SOURCE_DIR}/.git/hooks")
```

Comment:
It seems odd to me to do this in CMake. If the `pre-commit` hook now runs on Windows as well, why not install the hooks by default?

Reply:
Is there such thing as hooks which are installed by default? I thought they are local to the repository so they always need to be set up either manually or by some script.

Reply:
Looking carefully at the Git documentation, there's a note about this which I missed on my previous read:

> It’s important to note that client-side hooks are not copied when you clone a repository. If your intent with these scripts is to enforce a policy, you’ll probably want to do that on the server side [...]

[Source](https://git-scm.com/book/en/v2/Customizing-Git-Git-Hooks)

So it looks like you are correct.